### ISSUE TITLE
Gophish no longer accepting gmail source

### ISSUE THREAD

**Issue by loza0269** [gophish/gophish#1220]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: 
v0.4
Brief description of the issue: 
For over a year now I would create my phishing template through Gmail and then click Show Original, copy and paste that into the email template and it would be done. Now, I assume because of the updated Gmail client when copying into the template it says there was an error inserting into the database.

What are you expecting to see happen? :
The source is copied successfully and the template saving and appearing how I created it.

What are you seeing happen?
Error inserting template into database
If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:

Copied from gmail months ago:

```html
<html><head></head><body><div dir="ltr"><div dir="ltr">Hello {{.FirstName}},<div><br/></div><div>Please take a moment to learn about the impact of <a href="{{.URL}}">Hurricane Florence</a>. Any awareness or donations for this tragedy would be greatly appreciated and could help several families find food and shelter during the relief process.</div><div><br/></div><div>If you would like to make a donation, please visit:</div><div><a href="{{.URL}}"><br/></a></div><div><a href="{{.URL}}">https://www.redcross.org/donate/hurricane-florence-donations.html/</a></div><div><br/></div><div>Our thoughts and prayers are with those affected.</div><div>Thank you,</div><div><div><img src="cid:redcross.png" alt="image.png" width="125" height="42" style="margin-right: 0px;"/><br/></div></div></div></div>
{{.Tracker}}</body></html>

```
Copied from Gmail this morning:

```html
MIME-Version: 1.0
Date: Thu, 27 Sep 2018 10:37:45 -0400
Message-ID: <CAJrYEN+1E-Q3-QZHjzsQoYTKOq0bdezmr5j_OubUyqeh-cLs=g@mail.gmail.com>
Subject: Hurricane Florence Help
From: Christian Lozach <clozach627@gmail.com>
To: clozach@nbty.com
Content-Type: multipart/related; boundary="000000000000443ff90576db48ea"

--000000000000443ff90576db48ea
Content-Type: multipart/alternative; boundary="000000000000443ff60576db48e9"

--000000000000443ff60576db48e9
Content-Type: text/plain; charset="UTF-8"

Hello {{.FirstName}},

Were you or somebody you know affected by Hurricane Florence?

If so, you may apply for compensation for the damages received during the
tragedy. If you would like to *apply or refer a friend *click the link
below.

https://www.redcross.org/apply/dollars-for-damages-application.html/
<http://google.com>

Donations are also appreciated for those who were unaffected. If you would
like to donate click here <http://google.com>.

Our thoughts and prayers,
[image: redcross.png]

--000000000000443ff60576db48e9
Content-Type: text/html; charset="UTF-8"
Content-Transfer-Encoding: quoted-printable

<div dir=3D"ltr"><div dir=3D"ltr">Hello {{.FirstName}},<div><br></div><div>=
Were you or somebody you know affected by Hurricane Florence?=C2=A0</div><d=
iv><br></div><div>If so, you may apply for compensation for the damages rec=
eived during the tragedy. If you would like to <b>apply or refer a friend <=
/b>click the link below.</div><div><br></div><div><a href=3D"http://google.=
com">https://www.redcross.org/apply/dollars-for-damages-application.html/</=
a><br></div><div><br></div><div>Donations are also appreciated for those wh=
o were unaffected. If you would like to donate click <a href=3D"http://goog=
le.com">here</a>.</div><div><br></div><div>Our thoughts and prayers,</div><=
div><div><img src=3D"cid:ii_jmkoho6m0" alt=3D"redcross.png" width=3D"149" h=
eight=3D"50" style=3D"margin-right: 0px;"><br></div></div><div><br></div></=
div></div>

--000000000000443ff60576db48e9--
--000000000000443ff90576db48ea
Content-Type: image/png; name="redcross.png"
Content-Disposition: attachment; filename="redcross.png"
Content-Transfer-Encoding: base64
X-Attachment-Id: ii_jmkoho6m0
Content-ID: <ii_jmkoho6m0>


--000000000000443ff90576db48ea--
```
Please provide as many steps as you can to reproduce the problem:
 * Create an email through gmail
 * Send it to yourself
 * open the email and select the 3 vertical dots
 * select show original and copy to clipboard
 * import to email template
 
 Notes:
If anyone has any other methods they find useful when importing email templates, I'm always open to alternatives. For my purpose, I would just need to be able to have rich text and format inline images without coding HTML. 

Also, the one copied from Gmail months ago was imported as a template successfully, so it may be altered from whatever gophish inserts into the HTML (tracking image, etc)

Thank you for your time!

**Comment by jordan-wright** [comment]
Hi @loza0269,

I recommend upgrading Gophish to the latest version. I was able to import that email just fine in v0.7.1, so it might be an older bug that's causing the issue.

I hope this helps! Let me know if you continue to run into issues after upgrading and I'll be happy to reopen this issue to continue troubleshooting.

**Comment by loza0269** [comment]
Hi Jordan - thank you for the response. I upgraded to v0.7.1. Although, I am still receiving an error with this import. This time the behavior is a bit different. Before if I imported the email, gophish just couldn't make sense of it. Now, when I import, it looks as if it is successful with the formatting setup although when I actually click create template I still get the "error inserting template into database".
